src/components/Feed/HeroBanner.js

```diff
@@ -48,8 +48,8 @@ function HeroBanner({ onRequestNewProposal }) {
           position: relative;
           ${tabletMode &&
             `
-          min-height: ${27 * GU}px;
-          ${!compactMode && `border-radius: ${BIG_RADIUS}px;`}      
+          min-height: ${28 * GU}px;
```

Comment:
Should be 28.25 to have same height as Wrapper component

Reply:
<img width="356" alt="Screen Shot 2021-05-17 at 9 25 37 PM" src="https://user-images.githubusercontent.com/11763623/118572312-81578880-b756-11eb-8a2b-33bf4928deea.png">

<img width="417" alt="Screen Shot 2021-05-17 at 9 25 46 PM" src="https://user-images.githubusercontent.com/11763623/118572378-b06dfa00-b756-11eb-9d4e-40d34b67b371.png">

i think they are like that right?